etc/config.yaml.dist

```diff
@@ -1,26 +1,26 @@
 repository_url:
-  # [Changeable: init-project -c]
+  # [To apply changes: init-project -c]
```

Comment:
Actually should be `-fc`, `-c` by itself will not do anything at the moment. Maybe it make sense to change current behavior, but it might be tricky on Windows since it does not suppor NFS.